setup.cfg

```diff
@@ -6,19 +6,6 @@ skip = *.json,*.h,*.cpp,*.c,.riot,.tox,.mypy_cache,.git,*ddtrace/vendor,tests/co
 exclude-file = .codespellignorelines
 ignore-words-list = asend,dne,fo,medias,ment,nin,ot,setttings,statics,ba,spawnve,doas
 
-# DEV: We use `conftest.py` as a local pytest plugin to configure hooks for collection
-[tool:pytest]
-# --cov-report is intentionally empty else pytest-cov will default to generating a report
-addopts =
-  --cov=ddtrace/
-  --cov=tests/
-  --cov-append
-  --cov-report=
-  --durations=10
-  --junitxml=test-results/junit.xml
-# DEV: The default is `test_*\.py` which will miss `test.py` files
-python_files = test*\.py
-asyncio_mode = auto
```

Comment:
uhh is this necessary to remove? I think these are pretty important for our CI

Reply:
yea definitely will be restored when the PR is ready for a final pass